<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GeneratePasswordDialog</class>
 <widget class="QDialog" name="GeneratePasswordDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>151</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Generate Password Options</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <property name="whatsThis">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="bottomMargin">
    <number>4</number>
   </property>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="3" column="2">
      <widget class="QPushButton" name="genButton">
       <property name="text">
        <string>Generate</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QCheckBox" name="upperCheck">
       <property name="text">
        <string>Uppercase</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="passwordEdit"/>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="lengthBox">
       <property name="minimum">
        <number>8</number>
       </property>
       <property name="maximum">
        <number>256</number>
       </property>
       <property name="value">
        <number>12</number>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="lengthLabel">
       <property name="text">
        <string>Password Length:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QCheckBox" name="numberCheck">
       <property name="text">
        <string>Numbers</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QCheckBox" name="specialCheck">
       <property name="text">
        <string>Special Characters</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QCheckBox" name="lowerCheck">
       <property name="text">
        <string>Lowercase</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="passwordLabel">
       <property name="text">
        <string>Password:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item alignment="Qt::AlignRight">
    <widget class="QFrame" name="frame">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="copyButton">
        <property name="text">
         <string>Copy to Clipboard</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="okButton">
        <property name="text">
         <string>OK</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="closeButton">
        <property name="text">
         <string>Close</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>genButton</tabstop>
  <tabstop>copyButton</tabstop>
  <tabstop>okButton</tabstop>
  <tabstop>closeButton</tabstop>
  <tabstop>upperCheck</tabstop>
  <tabstop>numberCheck</tabstop>
  <tabstop>lowerCheck</tabstop>
  <tabstop>specialCheck</tabstop>
  <tabstop>lengthBox</tabstop>
  <tabstop>passwordEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
